### ISSUE TITLE
Downgrade from 0.8.x to version before 0.8.0-rc3 is broken

### ISSUE THREAD

**Issue by janeczku** [rancher/os#1744]
**RancherOS Version: (ros os version)**
0.8.1

`ros os upgrade` command in 0.8.1 invokes the installer with `installType=upgrade`. But the installer for versions before 0.8.0-rc3 (which uses the `laydown-os` script) expect the installType to be `rancher-upgrade`.

Reference:

https://github.com/rancher/os/blob/v0.8.1/cmd/control/os.go#L199
https://github.com/rancher/os/blob/v0.8.1/cmd/control/install.go#L173
https://github.com/rancher/os/blob/v0.7.1/scripts/installer/lay-down-os#L242

```
$ sudo ros os upgrade -f -i rancher/os:v0.7.1
Upgrading to rancher/os:v0.7.1
Pulling os-upgrade (rancher/os:v0.7.1)...
v0.7.1: Pulling from rancher/os
6bbedd9b76a4: Pull complete
fc19d60a83f1: Pull complete
de413bb911fd: Pull complete
2879a7ad3144: Pull complete
668604fde02e: Pull complete
c01770a76381: Pull complete
f51d1a0149a6: Pull complete
d82e4174c33b: Pull complete
Digest: sha256:35db9caf602754f2f32578d47d5822ddb5b02db8f5f524f9ffe6910e3c381bb6
Status: Downloaded newer image for rancher/os:v0.7.1
os-upgrade_1 | ++ dirname /scripts/lay-down-os
os-upgrade_1 | + SCRIPTS_DIR=/scripts
os-upgrade_1 | + VERSION=v0.7.1
os-upgrade_1 | + getopts i:f:c:d:t:r:o:p:ka: OPTION
os-upgrade_1 | + case ${OPTION} in
os-upgrade_1 | + ENV=upgrade
os-upgrade_1 | + getopts i:f:c:d:t:r:o:p:ka: OPTION
os-upgrade_1 | + case ${OPTION} in
os-upgrade_1 | + ROLLBACK_VERSION=v0.8.1
os-upgrade_1 | + getopts i:f:c:d:t:r:o:p:ka: OPTION
os-upgrade_1 | + [[ '' == \a\r\m ]]
os-upgrade_1 | + DIST=/dist
os-upgrade_1 | + CLOUD_CONFIG=/scripts/conf/empty.yml
os-upgrade_1 | + CONSOLE=tty0
os-upgrade_1 | + BASE_DIR=/mnt/new_img
os-upgrade_1 | + BOOT=boot/
os-upgrade_1 | + PARTITION=1
os-upgrade_1 | + KERNEL_ARGS='rancher.state.dev=LABEL=RANCHER_STATE rancher.state.wait console=tty0'
os-upgrade_1 | + '[' -n upgrade ']'
os-upgrade_1 | + case ${ENV} in
os-upgrade_1 | + echo 'upgrade is not a valid environment'
os-upgrade_1 | upgrade is not a valid environment
os-upgrade_1 | + exit 1
INFO[0008] Rebooting

```

**Comment by SvenDowideit** [comment]
yup - see #1743 for the first 2 parts of the fix for it :)

**Comment by SvenDowideit** [comment]
resolved in v0.9.2-rc4
